code/datums/spells/area_teleport.dm

```diff
@@ -19,6 +19,8 @@
 		A = pick(SSmapping.teleportlocs)
 
 	if(!A)
+		smoke_spread = 0
+		smoke_amt = 0
```

Comment:
This will do nothing on the first cast, but I assume it prevents this issue from cropping up on subsequent casts?